go.mod

```diff
@@ -0,0 +1,19 @@
+module marathon_exporter
```

Comment:
Can you propose a different PR to switch to go modules please?

Reply:
I just remove go modules.